17_piano_rolls.md

```diff
@@ -1,5 +0,0 @@
-# Nathan's Piano Rolls
```

Comment:
We both made this, I had appendix_a_piano_rolls.md previously.

I think this might be better as appendix f, I had made it appendix a so that Graham's would be appendix g .  B^D